application/artifacts/artifacts_view.py

```diff
@@ -69,21 +70,70 @@ def _find_multiple_by(params):
     return artifacts.order_by('created_at')[_from:_to]
 
 
-class ArtifactsController(ApplicationController):
-    """ Controller for Artifacts """
-
-    method_decorators = [check_es_connection]
+class ArtifactView(MethodResource):
+    """ Access Artifacts by id """
```

Comment:
Just a reminder: Docstrings start directly after the """ so you would have a space here which isn't good practice.
Obviously I don't expect you to change all of them, but if you change old docstrings it would be useful ;)

Reply:
Yeh I'm getting confused by old code and old docstrings I can change it project wide with a find and replace though.